.travis.yml

```diff
@@ -37,7 +37,7 @@ matrix:
         - SWIFTPM_TEST_Commandant=YES swift test
       env:
         - JOB=Linux
-        - SWIFT_VERSION=4.0-DEVELOPMENT-SNAPSHOT-2017-08-04-a
+        - SWIFT_VERSION=DEVELOPMENT-SNAPSHOT-2017-09-20-a
```

Comment:
Is there any reason not to use `4.0-RELEASE`?

Reply:
@norio-nomura Because that is tested above (L24-L33) with `.swift-version`.

Reply:
Ah, I see. I did not notice. Thanks.